handshake_client.go

```diff
@@ -119,7 +133,8 @@ func (c *Conn) clientHandshake() error {
 	var session *ClientSessionState
 	var cacheKey string
 	sessionCache := c.config.ClientSessionCache
-	if c.config.SessionTicketsDisabled {
+	// TLS 1.3 has no session resumption based on session tickets.
```

Comment:
Don't you mean the current client implementation does not support resumption?

Reply:
That's correct. Until early data support is implemented for the client, there is little reason to store the TLS 1.3 session ticket. Maybe I should modify this comment to be more accurate and reflect that.